crates/bevy_core/Cargo.toml

```diff
@@ -20,6 +20,10 @@ bevy_utils = { path = "../bevy_utils", version = "0.9.0-dev" }
 
 # other
 bytemuck = "1.5"
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Do you need the `derive` feature? it doesn't seem you use it.

Reply:
Good point, removed!